crates/agent/src/base.rs

```diff
@@ -25,6 +25,19 @@ use strata_btcio::rpc::{
 };
 use tracing::trace;
 
+pub(super) const MIN_RELAY_FEE: Amount = Amount::from_sat(5000);
```

Comment:
min relay fee is 1 sat/vByte
how this is 5k?
```suggestion
pub(super) const MIN_RELAY_FEE: Amount = Amount::from_sat(5_000);
```

Reply:
You can pretty much ignore changes to the `agent` crate. This is only used for local runs. This value was determined via hit and trial with default settings of the bitcoin node.